### ISSUE TITLE
keras to savedmodel: label names issue

### ISSUE THREAD

**Issue by EmnamoR** [tensorflow/serving#1234]
![screenshot from 2019-01-14 12-57-58](https://user-images.githubusercontent.com/23196931/51113023-f9b58400-1800-11e9-886f-3b038665e102.png)
### Problem description
I am trying to bring my models into production. 
I successfully converted keras models to tensorflow savedmodels for serving and deployed them.
Now i can run multiple models with multiple versions but the issue i cant have the label names in the model.outputs and this is how the predictions looks like in the image

any one can help me with this ? and thank you

**Comment by hgadig** [comment]
Did you check your model versions in model_config_file. Is it reflecting the version information ?
Also you can add model version labels to the model_config_file, and send inference requests with those version labels. In this case, you can use version labels instead of model versions. Please refer [this guide](https://github.com/tensorflow/serving/blob/master/tensorflow_serving/g3doc/serving_config.md) for more information.

**Comment by hgadig** [comment]
Closing this at it is in "awaiting response" status for more than a week. Feel free to add comments(if any), we will reopen the issue. Thanks !
